### ISSUE TITLE
Regarding the loading error of action.yaml

### ISSUE THREAD

**Issue by ghost** [de-vri-es/setup-git-credentials#28]
Hello,

I hope this message finds you well. I have been using `de-vri-es/setup-git-credentials@v2` action, but I have encountered the following error:

```
Error: System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values. (Parameter ''using: node20' is not supported, use 'docker', 'node12' or 'node16' instead.')
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load de-vri-es/setup-git-credentials/v2/action.yml
```

Even when I specify `@v2.1.0`, I still encounter the same error. 

Could you please take a look into this issue and provide assistance?

Thank you for your attention.

**Comment by de-vri-es** [comment]
Thanks for reporting the issue. This was recently introduces in `v2.1.1`.

I've fixed the node version to `node16` and released as `2.1.1`. The `v2` and `v2.1` tags are also updated.

Thanks for reporting it!

**Comment by smlgbl** [comment]
Strange - it works just fine when I run actions with this module directly from my fork.

**Comment by smlgbl** [comment]
@takuya-takahash1 Which type of github-runner are you running this on? It's working fine on ubuntu-latest

**Comment by ghost** [comment]
@smlgbl 
I cannot provide details, but I am running on a self-hosted runner using Image : _myoung34/github-runner:latest_ on EKS.

**Comment by ghost** [comment]
@de-vri-es Issue has been resolved. Thank you for your prompt assistance!

**Comment by de-vri-es** [comment]
No problem. However, I would prefer to update to `node20` again. Is that self hosted runner image outdated?

If it is, one option is to stick with `v2.0` if you can't support `node20`. Then `v2.1` can upgrade to `node20`.

**Comment by ghost** [comment]
@de-vri-es 
We are aware that the runner image is out of date and will be addressed in the near future. 
Until then, we will use `v2`, so you can upgrade `v2.1.0` to `node20`.

**Comment by de-vri-es** [comment]
Be sure to pin it to `v2.0`, because the `v2` tag will update to point to the latest `v2.x.y`.

**Comment by ghost** [comment]
@de-vri-es 
I understand. We will modify the `v2` we are using now to `v2.0`. 
Thank you.

**Comment by de-vri-es** [comment]
Cool! Upgraded to `node20` again and released as `v2.1.2`.

I also added a note to the readme to list the `node20` requirement and the workaround for custom runners. :)
